config/initializers/doorkeeper.rb

```diff
@@ -49,15 +54,27 @@
   # Change the way access token is authenticated from the request object.
   # By default it retrieves first from the `HTTP_AUTHORIZATION` header, then
   # falls back to the `:access_token` or `:bearer_token` params from the `params` object.
-  # Check out the wiki for mor information on customization
+  # Check out the wiki for more information on customization
   # access_token_methods :from_bearer_authorization, :from_access_token_param, :from_bearer_param
 
-  # Change the test redirect uri for client apps
+  # Change the native redirect uri for client apps
   # When clients register with the following redirect uri, they won't be redirected to any server and the authorization code will be displayed within the provider
   # The value can be any string. Use nil to disable this feature. When disabled, clients must provide a valid URL
   # (Similar behaviour: https://developers.google.com/accounts/docs/OAuth2InstalledApp#choosingredirecturi)
   #
-  # test_redirect_uri 'urn:ietf:wg:oauth:2.0:oob'
+  # native_redirect_uri 'urn:ietf:wg:oauth:2.0:oob'
+
```

Comment:
Where did blocks like the following go?  (want to make sure we are still authenticating and restricting access appropriately):

```ruby
  # This block will be called to check whether the resource owner is authenticated or not.
  resource_owner_authenticator do
    authenticate_user!
    current_user
  end

  # If you want to restrict access to the web interface for adding oauth authorized applications, you need to declare the block below.

  # We allow all users of Accounts to manage applications
  # We subclassed Doorkeeper::ApplicationsController to provide better
  # control over access to the Doorkeeper::Application pages
  admin_authenticator do
    authenticate_user!
    current_user
  end
```